openpype/hosts/maya/plugins/publish/validate_animated_reference.py

```diff
@@ -0,0 +1,63 @@
+import pyblish.api
+from openpype.pipeline.publish import (
+    PublishValidationError,
+    ValidateContentsOrder
+)
+from maya import cmds
+
+
+class ValidateAnimatedReferenceRig(pyblish.api.InstancePlugin):
+    """
+        Validate all the nodes underneath skeleton_Anim_SET
+        should be reference nodes
+    """
+
+    order = ValidateContentsOrder
+    hosts = ["maya"]
+    families = ["animation.fbx"]
+    label = "Animated Reference Rig"
+    accepted_controllers = ["transform", "locator"]
+
+    def process(self, instance):
+        animated_sets = instance.data["animated_skeleton"]
+        if not animated_sets:
+            self.log.debug(
+                "No nodes found in skeleton_Anim_SET..Skipping..")
```

Comment:
```suggestion
                "No nodes found in skeletonAnim_SET. Skipping..."
            )
```
Make sure to always use the `...` when you're using something like `..` or `....`. Best to be consistent.